release/src/slack.ts

```diff
@@ -169,12 +169,25 @@ export function sendSlackMessage({ channelName = SLACK_CHANNEL_NAME, message }:
   });
 }
 
-async function getSlackChannelId(channelName: string) {
+async function getSlackChannelId(
+  channelName: string,
+  cursor?: string,
+): Promise<string | undefined> {
```

Comment:
This is the type we're currently returning, I just set it here just to make it clear.